### PR TITLE
Add missing output links

### PR DESCRIPTION
**Pull request by EndymionJkb**
# Description

Noticed there were a few missing output links in /deployments. Also some obsolete mentions of kovan, which were replaced with goerli. Finally, made the working/order consistent.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [X] Documentation or wording changes
- [ ] Other

## Checklist:

- [X] The diff is legible and has no extraneous changes
- [N/A] Complex code has been commented, including external interfaces
- [N/A] Tests are included for all code paths
- [X] The base branch is either `master`, or there's a description of how to merge

### CODE REVIEW

File: pkg/deployments/tasks/20220707-distribution-scheduler/readme.md [hunk 1]
```diff
@@ -5,5 +5,6 @@ Deployment of the `DistributionScheduler`, which can be used to schedule token d
 ## Useful Files
 
 - [Ethereum mainnet addresses](./output/mainnet.json)
+- [Avalanche mainnet addresses](./output/avalanche.json)
```
Comment (jubeira) [inline pkg/deployments/tasks/20220707-distribution-scheduler/readme.md:8]: Hmm the entry here is correct, but the fact that it is not deployed in other L2s makes me think that this is probably not required in Avalanche, and it shouldn't be considered a 'canonical' deployment.

**Review by jubeira — APPROVED** [review]
Just one side comment; otherwise LGTM.

Isn't there a way to add this to CI? (i.e. check that all outputs have a link in the readme)

File: pkg/deployments/tasks/20220707-distribution-scheduler/readme.md (hunk 1 above)
Comment (EndymionJkb) [reply pkg/deployments/tasks/20220707-distribution-scheduler/readme.md:8]: Yes; I'm not sure why that was deployed there. (It wasn't one of the recent ones I just did.) Anyway not hurting anything.

**Comment by EndymionJkb** [comment]
> Isn't there a way to add this to CI? (i.e. check that all outputs have a link in the readme)

Well, the problem is our two-stage deployment. The artifacts always have to be there: and that is checked. But the outputs will only be there in the actual Deployment PR, not the Deployment Preparation PR, so we'd have to know somehow which one we were in to do the check, and I don't think we should have "logic" like that in CI.

It's on the checklist; we just have to be more vigilant. (Most of these are from before we had the checklist in place.)

**Pull request merged**